pyth-sdk-solana/examples/eth_price.rs

```diff
@@ -19,8 +19,8 @@ fn main() {
 
     loop {
         // get price data from key
-        let eth_price_data = clnt.get_account_data(&eth_price_key).unwrap();
-        let eth_price = load_price(&eth_price_data).unwrap();
+        let mut eth_price_account = clnt.get_account(&eth_price_key).unwrap();
+        let eth_price = load_price_from_account(&eth_price_key, &mut eth_price_account).unwrap();
```

Comment:
Should the Solana SDK expose a convenience function wrapping this, which given a client and a price key, returns a price? Probably a nice to have though 😄

Reply:
If we get involved with client in the sdk then it requires a lot of dirty features to handle compiling in bpf. So essentially let's say for now it's sdk not a client. Maybe in the future we can consider a separate client package.